src/Sulu/Bundle/ContactBundle/Import/Import.php

```diff
@@ -226,10 +234,10 @@ class Import
      * @var array
      */
     protected $accountTypeMappings = array(
-        Account::TYPE_BASIC => '',
-        Account::TYPE_LEAD => 'lead',
-        Account::TYPE_CUSTOMER => 'customer',
-        Account::TYPE_SUPPLIER => 'supplier',
+        '' => Account::TYPE_BASIC,
```

Comment:
is there a possiblity to not use an empty string as a key?

e.g. change the ORM mapping so that the default key is "basic"?